File: examples/java/walkerfx/src/main/java/be/kuleuven/walkerfx/model/Walker.java
package be.kuleuven.walkerfx.model;

public class Walker {
    private int x;
    private int y;

    private static final int MODIFIER = 5;

    public Walker(int x, int y) {
        this.x = x;
        this.y = y;
    }

    public int getX() {
        return x;
    }

    public int getY() {
        return y;
    }

    public void goLeft() {
        x-= MODIFIER;
    }

    public void goRight() {
        x+= MODIFIER;
    }

    public void goUp() {
        y-= MODIFIER;
    }

    public void goDown() {  y+= MODIFIER; }
}

File: examples/kotlin/walkerfx/src/main/kotlin/be/kuleuven/walkerfx/model/Walker.kt
package be.kuleuven.walkerfx.model

class Walker(x: Int, y: Int) {
    var x = x
        private set
    var y = y
        private set
    val MODIFIER = 5

    fun goUp() { y -= MODIFIER }
    fun goDown() { y += MODIFIER }
    fun goLeft() { x -= MODIFIER }
    fun goRight() { x += MODIFIER }
}